backend/src/handlers/programs_handler.go

```diff
@@ -11,6 +11,8 @@ func (srv *Server) registerProgramsRoutes() []routeDef {
 	axx := []models.FeatureAccess{models.ProgramAccess}
 	return []routeDef{
 		{"GET /api/programs", srv.handleIndexPrograms, false, axx},
+		{"GET /api/programs-overview-stats", srv.handleIndexProgramsFacilitiesStats, true, axx},
+		{"GET /api/programs-overview-table", srv.handleIndexProgramsOverviewTable, true, axx},
```

Comment:
@calisio It feels like these might be too coupled to the front-end especially when naming it closely to how it'll be presented. What do you think about these?

`/api/programs/stats` or `api/programs/summary` for `/api/programs-overview-stats` 
and
`/api/programs/detailed-list` for `/api/programs-overview-table`